ui/app/styles/components/replication-dashboard.scss

```diff
@@ -15,7 +15,13 @@
     margin-bottom: $spacing-xs;
   }
 
+  .reindexing-alert,
+  .syncing-alert {
+    margin-top: 2rem;
```

Comment:
I'm totally fine with this being 2rem, but would anything in the helpers.scss file work, e.g. 
```
.has-top-margin-xl {
  margin-top: $spacing-xl;
}
```

Maybe not use the class has-top-margin-xl, but replace the 2rem with $spacing-xl?

Reply:
good call, i'll use $spacing-xl. thank you!